src/apps/states/s3StorageRepo/getStateContent.ts

```diff
@@ -7,18 +9,12 @@ export default (config: Config) => {
   return async (opts: GetStateContentOptions): Promise<GetStateContentResult> => {
     const dir = getStorageDir({ subfolder: config.subFolder, lrs_id: opts.lrs_id });
     const filePath = `${dir}/${opts.key}`;
-    await config.client
-      .getObject({
-        Bucket: config.bucketName,
-        Key: filePath,
-      })
-      .promise();
-    const content = config.client
-      .getObject({
-        Bucket: config.bucketName,
-        Key: filePath,
-      })
-      .createReadStream();
-    return { content };
+
+    const getObjectCommand = new GetObjectCommand({
+      Bucket: config.bucketName,
+      Key: filePath,
+    });
+    const { Body } = await config.client.send(getObjectCommand);
+    return { content: Body as ReadableStream };
```

Comment:
Not sure if original code was meant to be (1) set header object (2) get object, was 2 get object calls.

Reply:
First call updated to HeadObject to check for existence (to be inline with usages elsewhere)